plugins/inputs/x509_cert/x509_cert.go

```diff
@@ -187,10 +188,39 @@ func getTags(cert *x509.Certificate, location string) map[string]string {
 	return tags
 }
 
+// copied from plugins/inputs/file/file.go
+func (c *X509Cert) refreshFilePaths() error {
```

Comment:
Let's add this logic to an `func (c *X509Cert) Init() error` function, it will only be ran once on plugin setup.  We can setup the globpath once, and then in `Gather()` we will call `g.Match()` each time.

Reply:
Hi, sorry I've been busy.

I've tried to match your request with 19441a29738b85f7bbd9f9e741636b6aba938c78